Scripts/Tracking/Follow/Modifier/ModifyTransform.cs

```diff
@@ -12,44 +12,29 @@ public class ModifyTransform : FollowModifier
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method